grunt_tasks/webpack.config.js

```diff
@@ -189,15 +189,15 @@ module.exports = {
             //   possible, it's not recommended. Try to require the original source
             //   to get better results.
             // This needs fixing later, as Webpack works better when provided with source.
-            // /node_modules\/jade/,
+            // /node_modules\/pug/,
             // /node_modules\/remarkable/
         ]
     },
     resolve: {
         alias: {
             'girder': paths.web_src
         },
-        extensions: ['.styl', '.css', '.jade', '.js', ''],
+        extensions: ['.styl', '.css', '.pug', '.jade', '.js', ''],
```

Comment:
I had initially removed it, but then I added it back in case plugins still wanted to use the .jade extension. I don't have a strong preference one way or the other, though.